File: Faltantes o por revisar/CleaningRoom.java
// java code to find whether 
// house can be cleaned or not 
import java.util.ArrayList;

public class CleaningRoom {

    static class Pair {

        int a, b;

        public Pair(int a, int b) {
            this.a = a;
            this.b = b;
        }
    }
    // Matrix A stores the string 
    static int A[][] = new int[105][105];
    // ans stores the pair of indices 
    // to be cleaned by the machine 
    static ArrayList<Pair> ans = new ArrayList<>();
    // Function for printing the 
    // vector of pair 

    static void print() {
        System.out.println("Yes, the house can be cleaned.");
        for (int i = 0; i < ans.size(); i++) {
            System.out.println((ans.get(i).a + " " + ans.get(i).b));
        }
    }
    // Function performing calculations 

    static void solve(int n) {
        // push every first cell in 
        // each row containing '.' 
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                if (A[i][j] == '.') {
                    ans.add(new Pair(i + 1, j + 1));
                    break;
                }
            }
        }
        // If total number of cells are 
        // n then house can be cleaned 
        if (ans.size() == n) {
            print();
            return;
        }

        ans = new ArrayList<>();

        // push every first cell in 
        // each column containing '.' 
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                if (A[i][j] == '.') {
                    ans.add(new Pair(i + 1, j + 1));
                    break;
                }
            }
        }
        // If total number of cells are 
        // n then house can be cleaned 
        if (ans.size() == n) {
            print();
            return;
        }
        System.out.println("house cannot be cleaned.");
    }

    public static void main(String[] args) {
        int n = 3;
        String s = "";
        s += ".**";
        s += ".**";
        s += ".**";
        int k = 0;
        //Loop to insert letters from 
        //string to array 
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                A[i][j] = s.charAt(k);
                k += 1;
            }
        }
        solve(n);
    }

}

File: Recien traducido/Geometria/CleaningRoom.py
# Python3 code to find whether 
# house can be cleaned or not 
  
# Matrix A stores the string 
A = [[0 for i in range(105)] for j in range(105)] 
  
# ans stores the pair of indices 
# to be cleaned by the machine 
ans = [] 
  
# Function for printing the 
# vector of pair 
def printt(): 
      
    print("Yes, the house can be cleaned.") 
    for i in range(len(ans)): 
        print(ans[i][0], ans[i][1]) 
          
# Function performing calculations 
def solve(n): 
    global ans 
      
    # push every first cell in 
    # each row containing '.' 
    for i in range(n): 
        for j in range(n): 
            if (A[i][j] == '.'): 
                ans.append([i + 1, j + 1]) 
                break
              
    # If total number of cells are 
    # n then house can be cleaned 
    if (len(ans) == n): 
        printt() 
        return 0
          
    ans = [] 
      
    # push every first cell in 
    # each column containing '.' 
    for i in range(n): 
        for j in range(n): 
            if (A[j][i] == '.'): 
                ans.append([i + 1, j + 1]) 
                break
              
    # If total number of cells are 
    # n then house can be cleaned 
    if (len(ans) == n): 
        printt() 
        return 0
    print("house cannot be cleaned.") 
  

n = 3
s = "" 
s += ".**"
s += ".**"
s += ".**"
k = 0
  
# Loop to insert letters from 
# string to array 
for i in range(n): 
    for j in range(n): 
        A[i][j] = s[k] 
        k += 1
  
solve(n) 
